content/docs/command-reference/run.md

```diff
@@ -161,8 +161,8 @@ data pipeline (e.g. random numbers, time functions, hardware dependency, etc.)
   determined by the logic described in the `-f` option) without asking for
   confirmation.
 
-- `--ignore-build-cache` - forcefully execute the `command` again, even if the
-  same `dvc run` command has already been run in this workspace. Useful if the
+- `--no-run-cache` - forcefully execute the `command` again, even if the same
+  `dvc run` command has already been run in this workspace. Useful if the
```

Comment:
it is not this workspace anymore, right?

Reply:
Yeah, but I have trouble explaining it here better without defining run-cache before that. Looks like I should create `run-cache` entry in our user guide to explain it. But `--run-cache` is hidden in `push/pull/etc` right now, so we can keep `workspace` in this explanation, as without those options you can't really transfer run-cache anywhere.

Reply:
yeah, make it a checkbox in the ticket to write and update stuff related to the run cache.